resolwe_bio/descriptors/unified-descriptor.yml

```diff
@@ -0,0 +1,1537 @@
+              required: false
+            - name: molecule
+              label: Library made from / Molecule
+              type: basic:string
+              description: |
+                Select what molecule your samples are from, e.g. totalRNA, polyA RNA, genomic DNA, etc.
+              required: false
+              choices:
+                - label: Total RNA
+                  value: total_rna
+                - label: polyA RNA
+                  value: polya_rna
+                - label: Cytoplasmic RNA
+                  value: cytoplasmic_rna
+                - label: Nuclear RNA
+                  value: nuclear_rna
+                - label: Genomic DNA
+                  value: genomic_dna
+                - label: Protein
+                  value: protein
+                - label: Other
+                  value: other
+            - name: library_depleted_in
+              label: Library depleted in
+              type: basic:string
+              description: |
+                Select how your library was depleted, otherwise choose N/A.
+              required: false
+              choices:
+                - label: rRNA
+                  value: rrna
+                - label: polyA mRNA
+                  value: polya_mrna
+                - label: N/A
+                  value: na
+            - name: library_treatment
+              label: Library treatment
+              type: basic:string
+              description: Describe how your library was treated.
+              required: false
+            - name: replication_type
+              label: Replication type
+              type: basic:string
+              description: |
+                Select what replication type you used. For more info, go to
+                https://www.encodeproject.org/data-standards/terms/.
+              required: false
+              choices:
+                - label: Biological
+                  value: biological
+                - label: Isogenic
+                  value: isogenic
+                - label: Technical
+                  value: technical
+                - label: Sequencing
+                  value: sequencing
+                - label: Pseudoreplicate
+                  value: pseudoreplicate
+                - label: Other
+                  value: other
+            - name: platform
+              label: Platform
+              type: basic:string
+              description: Select the platform used to sequence your samples.
+              required: false
+              choices:
+                - label: NextSeq 500
+                  value: nextseq_500
+                - label: HiSeq 2500
+                  value: hiseq_2500
+                - label: HiSeq 2000
+                  value: hiseq_2000
+                - label: NovaSeq 6000
+                  value: novaseq_6000
+                - label: Other
+                  value: other
+            # - name: target_of_chip_assay
```

Comment:
We disabled `ChIP-seq` but can enabled later. 
@jkokosar

Reply:
ChIP-seq fields were removed. Changelog was updated accordingly.